bin/reth/src/t8n/provider.rs

```diff
@@ -0,0 +1,42 @@
+#![allow(unreachable_pub)] // TODO: Remove.
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+
+use reth_primitives::{Address, Bytes, H256, U256, U64};
+
+#[derive(Serialize, Deserialize, PartialEq, Eq)]
+#[serde(rename_all = "camelCase")]
+pub(crate) struct PrestateAccount {
```

Comment:
Effectively the same as `GenesisAccount` in `reth-primitives` (minus storage slots and code)